clients/src/main/resources/common/message/VoteRequest.json

```diff
@@ -18,30 +18,36 @@
   "type": "request",
   "listeners": ["controller"],
   "name": "VoteRequest",
-  "validVersions": "0",
+  "validVersions": "0-1",
```

Comment:
Please add a comment to mention what changed in v.1.

Reply:
Fixed.